test/test_on_gpu/test_torch_model.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import numpy as np
+import torch
+import pytest
+
+from gluonts.dataset.pandas import PandasDataset
+from gluonts.evaluation import make_evaluation_predictions
+from gluonts.torch.model.estimator import PyTorchLightningEstimator
+from gluonts.torch.model.simple_feedforward import SimpleFeedForwardEstimator
+from gluonts.torch.model.deepar import DeepAREstimator
+from gluonts.evaluation import Evaluator
+
+
+@pytest.mark.parametrize(
+    "torch_model_cls",
+    [
+        SimpleFeedForwardEstimator,
+        DeepAREstimator
+    ]
+)
+def test_torch_model_on_gpu(
+    torch_model_cls: torch.model.PyTorchLightningEstimator
```

Comment:
You're passing a class now (so, a `Type` and not a `PyTorchLightningEstimator`), but you could indeed pass the estimator object directly

Reply:
What is the preferred way in this case?

Reply:
I think passing the estiamator object (so, creating it as part of the `pytest.mark.parametrize` decorator)